packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -322,55 +325,51 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
     <Button icon={<CalendarIcon />} title={props.openCalendarTitle} iconOnly disabled={props.disabled} />
   );
 
-  const element = (
-    <Ref innerRef={datepickerRef}>
-      {getA11yProps.unstable_wrapWithFocusZone(
-        <ElementType
-          {...getA11yProps('root', {
-            className: classes.root,
-            ...unhandledProps,
-          })}
-        >
-          {!props.buttonOnly &&
-            createShorthand(Input, input, {
-              defaultProps: () =>
-                getA11yProps('input', {
-                  placeholder: props.inputPlaceholder,
-                  disabled: props.disabled,
-                  error: !!error,
-                  value: formattedDate,
-                  readOnly: !allowManualInput,
-                  required: props.required,
-                  ref: inputRef,
-                  'aria-label': formatRestrictedInput(restrictedDatesOptions, dateFormatting),
-                }),
-              overrideProps: overrideInputProps,
-            })}
-          {createShorthand(Popup, popup, {
-            defaultProps: () => ({
-              open: openState && !props.disabled,
-              content: calendarElement,
-              trapFocus: {
-                disableFirstFocus: true,
-              },
-              trigger: triggerButtonElement,
-              target: !props.buttonOnly ? inputRef.current : null,
-              position: 'below' as const,
-              align: 'start' as const,
+  const element = getA11yProps.unstable_wrapWithFocusZone(
+    <ElementType
+      {...getA11yProps('root', {
+        className: classes.root,
+        ...unhandledProps,
+      })}
+    >
+      {!props.buttonOnly &&
+        createShorthand(Input, input, {
+          defaultProps: () =>
+            getA11yProps('input', {
+              placeholder: props.inputPlaceholder,
```

Comment:
```suggestion
              placeholder: props.inputPlaceholder,
```

Should be this removed? It can be defined via shorthand `<Datepicker input={{ placeholder }} />`

Reply:
There is no specific reason for keeping the placeholder explicit. Will switch.

FYI: @pompomon

Reply:
I checked out the implementation again and the placeholder is part of the datepicker localization strings (ICalendarStrings). The user then always has the translations in one spot. 

Does it then from this stance make sense to keep the `inputPlaceholder` prop as is?

Reply:
Let's keep it as it is for now.